### PR TITLE
add IDE linting

### PR DESCRIPTION
**Pull request by lloydbenson**

### CODE REVIEW

**Review by Nargonath — APPROVED** [review]
LGTM, works on VSCode. We'll keep the @babel dep out for now and we can always add them back afterwards.

File: package.json [hunk 1]
```diff
@@ -14,12 +14,18 @@
...
+            "plugin:@hapi/module"
+        ]
+    },
     "dependencies": {
         "@hapi/boom": "9.x.x",
         "@hapi/hoek": "9.x.x"
     },
     "devDependencies": {
         "@hapi/code": "8.x.x",
+        "@hapi/eslint-plugin": "^5.1.0",
```
Comment (Marsup) [inline package.json:28]: Shouldn't it be kept as `"@hapi/eslint-plugin": "*",` so that it always follows lab's?

Comment (Nargonath) [reply package.json:28]: Good call @Marsup. I agree.

**Comment by lloydbenson** [comment]
Changed this to *. I appreciate the reviews since we want to do the same thing for all the modules. Let me know if any further changes are needed.

**Review by Nargonath — APPROVED** [review]

**Pull request merged**